### PR TITLE
Team2 tund20 implement extent report

### PR DESCRIPTION
**Pull request by tund-hcmue**
- create new class ExtentManager
- create new class ExtentTestManager
- create new class ReportListener
- add new method waitForElementToBeClickable in class WebKeyword
- Edit keyword setValueForElement used waitForElementToBeClickable

### CODE REVIEW

File: demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public WebKeyword(WebDriver driver) {
      * @return WebKeyword to set value for element
      */
     public WebKeyword setValueForElement(WebElement webElement, chooseTypeOfSelect type, String value){
```
Comment (TrangDao25399) [inline demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java]: rename to setValueForSelectElement or setValueForDdlElement

Comment (TrangDao25399) [inline demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java]: I think you should rename to setValueForSelectElement or setValueForDdlElement

Comment (tund-hcmue) [reply demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java]: thanks for feedback, i was rename it.

**Review by TrangDao25399 — APPROVED** [review]
I think it is good

**Comment by herrynhat** [comment]
Good Idea

**Pull request closed without merging**